### PR TITLE
Issue #12641: adds tests for design profile

### PR DESCRIPTION
**Pull request by rnveach**
Issue #12641

No regression needed.

### CODE REVIEW

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/design/throwscount/InputThrowsCount4.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+package com.puppycrawl.tools.checkstyle.checks.design.throwscount;
+
+import java.awt.AWTException;
+import java.io.EOFException;
+import java.io.FileNotFoundException;
+import java.sql.SQLException;
+
+public class InputThrowsCount4 {
+    void method1() throws Exception // violation
```
Comment (nrmancuso) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/design/throwscount/InputThrowsCount4.java:17]: Please add violation messages

**Review by nrmancuso — CHANGES_REQUESTED** [review]
Item:

**Review by romani — APPROVED** [review]
ok to merge as above is addressed

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/design/throwscount/InputThrowsCount4.java (hunk 1 above)
Comment (rnveach) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/design/throwscount/InputThrowsCount4.java]: Done.

**Review by nrmancuso — APPROVED** [review]

**Comment by nrmancuso** [comment]
Failure to download dependency in semaphore is not related to this PR. https://checkstyle.semaphoreci.com/jobs/accbe8ca-f87f-41dc-a35c-091b59472f0d

**Pull request merged**